src/cf-core/usage.md

```diff
@@ -86,6 +93,7 @@ Overwrite them in your own project by duplicating the variable `@key: value`.
 @size-iv:             18px; // h4-size
 @size-v:              14px; // h5-site
 @size-vi:             12px; // h6-size
+@size-code:           13px; // Custom size only for Mono code blocks
```

Comment:
What is this referencing?

Reply:
it's for the styling of the font in code snippets. only used on capital framework docs.